source/shared/core_stmt.cpp

```diff
@@ -552,7 +559,7 @@ void core_sqlsrv_bind_param( _Inout_ sqlsrv_stmt* stmt, _In_ SQLUSMALLINT param_
                 // avoid this silent truncation, we set the column_size to be "MAX" size for 
                 // string types. This will guarantee that there is no silent truncation for 
                 // output parameters.
-                if( direction == SQL_PARAM_OUTPUT ) {
+                if( direction == SQL_PARAM_OUTPUT && !stmt->conn->ce_option.enabled ) {
```

Comment:
please add a comment to explain why it's not necessary to set column_size here when column encryption is enabled